package.json

```diff
@@ -42,7 +42,7 @@
     "ember-maybe-import-regenerator": "^0.1.6",
     "ember-qunit": "^3.4.1",
     "ember-resolver": "^5.0.1",
-    "ember-simple-leaflet-maps": "^0.0.2",
+    "ember-simple-leaflet-maps": "^0.0.3",
```

Comment:
We should probably make this use 1.0.0, no real reason not to leverage semver (we won't run out of integers 😸).

Reply:
lol, yes, you're right. I'll open an issue.